content/en/docs/configuration/acme/_index.md

```diff
@@ -83,17 +83,25 @@ external account such as a CA custom database. This is typically not needed for
 most cert-manager users unless you know it is explicitly needed.
 
 External Account Bindings require three fields on an ACME `Issuer` which
-represents your ACME account. These fields are; the `keyID` of which your
-external account binding is indexed by the external account manager,
-`keySecretRef` which references a secret containing a base 64 encoded URL
-string of your external account symmetric MAC key, and finally `keyAlgorithm`,
-the MAC algorithm used to sign the JSON web string containing your External
-Account Binding when registering the account with the ACME server.
-
-> Note: The command `base64` is useful for encoding your MAC key if it is not
-> already `$ echo 'my-secret-key' | base64`, you can then create the Secret
-> resource with: `kubectl create secret generic eab-secret --from-literal
-> secret={base64 encoded secret key}`
+represents your ACME account. These fields are:
+
+- `keyID` - the key ID of which your external account binding is indexed by the
+external account manager
+- `keySecretRef` - the name and key of a secret containing a base 64 encoded 
+URL string of your external account symmetric MAC key
+- `keyAlgorithm` - the MAC algorithm used to sign the JSON web string 
+containing your External Account Binding when registering the account with the
+ACME server
+
+> Note: In _most_ cases, the MAC key must be base64URL-encoded. The following 
+> command will base64-encode a key and convert it to base64URL:
+> 
+>     $ echo 'my-secret-key' | base64 | sed -e 's/+/-/g' -e 's///_/g' -e 's/=//g'
+> 
+> You can then create the Secret resource with: 
+> 
+>     kubectl create secret generic eab-secret --from-literal \
```

Comment:
nit: Add a `$` here as well, just for consistency

Reply:
Fixed